particles/Tutorial/Javascript/README.md

```diff
@@ -0,0 +1,52 @@
+Welcome to developing using Arcs! By using Arcs as the platform for your system, you are choosing to value user sovereignty and leverage AI.  To get started, please obtain the project through [Github](https://github.com/PolymerLabs/arcs) using the instructions available there. For debugging purposes, we highly recommend having the Chrome Dev Tools also installed.
+
+Before we get to code, it is important that we are all speaking the same language, so we present two definitions to start.
+
+>Particle - Modular component of functionality. Ideally small units so particles can be interchangeable. 
```

Comment:
I'm sorry for being a horrible nitpicker...

Maybe use a list instead?

- _Particle_ - Modular component of functionality. Ideally small units so particles can be interchangeable. 
- _Recipe_ - A combination of particles to create features and systems.